src/components/numbers/Numbers.js

```diff
@@ -1,19 +1,23 @@
 import './numbers.scss';
 
 const Numbers = (props) => {
-  const { from } = props;
-  const { to } = props;
-  const { even } = props;
+  const { from, to, even } = props;
   const arr = [];
 
-  for (let num = from; num <= to; num++) {
+  for (let num = +from; num <= +to; num++) {
     arr.push(num);
     num++;
   }
 
-  return even
-    ? <ul className="numbers">{arr.filter(item => (item % 2 === 0)).map(item => <li>{item}</li>)}</ul>
-    : <ul className="numbers">{arr.filter(item => (item % 2)).map(item => <li>{item}</li>)}</ul>;
+  return (
+    <ul className="numbers">
+      {
+        even
+        ? arr.filter(item => (item % 2 === 0)).map(item => <li>{item}</li>)
+        : arr.filter(item => (item % 2)).map(item => <li>{item}</li>)
```

Comment:
`item => <li>{item}</li>` this part should be defined as a separate variable (do not repeat it)